### PR TITLE
Air viewer sireesha  2 to master for issue 28 : Created a pop up to ask for a correct file if wrong format selected.

### LINKED ISSUE #28: As a developer I want to create a pop-up to ask for a correct file, if wrong format file is selected.

### PR DESCRIPTION
**Pull request by sireeshanalabothu**

### CODE REVIEW

**Review by aanuhya31 — APPROVED** [review]
changes are reviewed looks fine to me.

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 1]
```diff
@@ -182,37 +182,32 @@ public class AIRViewerController implements Initializable {
...
-					file = fileChooser.showOpenDialog(stage);
-					path = file.getCanonicalPath();
-				}
-				loadedModel = new AIRViewerModel(Paths.get(path));
+        AIRViewerModel loadedModel = null;
+        try {
+            FileChooser fileChooser = new FileChooser();
+            fileChooser.setTitle("Open PDF File");
+            fileChooser.setInitialFileName(startPath);
+            Stage stage = (Stage) pagination.getScene().getWindow();
```
Comment (paidikondala) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: I see that this line of code is not synced with master. Can you please sync your changes with master

**Review by paidikondala — CHANGES_REQUESTED** [review]

File: src/main/java/edu/wright/airviewer2/AIRViewerController.java [hunk 2]
```diff
@@ -182,37 +182,32 @@ public class AIRViewerController implements Initializable {
...
+            File file = fileChooser.showOpenDialog(stage);
+            if (null != file) {
+                String path = file.getCanonicalPath();
```
Comment (paidikondala) [inline src/main/java/edu/wright/airviewer2/AIRViewerController.java]: Also, please sync this line of code with master

**Review by paidikondala — CHANGES_REQUESTED** [review]

**Review by paidikondala — APPROVED** [review]
Looks good now.

**Pull request merged**